doc/whats_new/v1.3.rst

```diff
@@ -236,6 +236,14 @@ Changelog
 - |Enhancement| Added the parameter `fill_value` to :class:`impute.IterativeImputer`.
   :pr:`25232` by :user:`Thijs van Weezel <ValueInvestorThijs>`.
 
+:mod:`sklearn.gaussian_process`
+................................
+
+- |API| Changing default behavior of :func:`sample_y` for its `random_state` from `0` to
+  `None`.
+  Updated `test_gpr.py` so that tests are deterministic.
+  :pr:`25789` by :user:`Kevin Ingles <ominusliticus>`.
```

Comment:
The default have not changed yet and will change in 1.5. Also, the change log is user facing, so they are informed of public API changes. We do not need to include changes about our tests. Here is my suggestion:

```suggestion
- |API| The default value for `random_state` in
  :meth:`GaussianProcessRegressor.sample_y` will change from `0` to `None` in v1.5.
  :pr:`25789` by :user:`Kevin Ingles <ominusliticus>`.
```